### PR TITLE
separates malf ai from traitor dynamic ruleset

### PR DESCRIPTION
**Pull request by Fikou**
## About The Pull Request

requested by head admins
separates malf ai from traitor so admins can treat it separately

## Why It's Good For The Game

so admins can set the weight instead of basing on traitor

## Changelog
:cl:
admin: separates malf ai from traitor dynamic ruleset
/:cl:

### CODE REVIEW

**Comment by dragomagol** [comment]
Does this fix #39573?

**Comment by SkeletalElite** [comment]
> Does this fix #39573?

unlikely since it's only touches dynamic implementation

**Comment by Jared-Fogle** [comment]
That's a dynamic issue. The answer is yes, I believe.

File: code/game/gamemodes/dynamic/dynamic_rulesets_roundstart.dm [hunk 1]
```diff
@@ -522,6 +522,32 @@
...
+	exclusive_roles = list("AI")
+	minimum_required_age = 0
+	required_candidates = 1
+	weight = 5
+	cost = 10
+	requirements = list(10,10,10,10,10,10,10,10,10,10)
+
+/datum/dynamic_ruleset/roundstart/malf/pre_execute()
+	. = ..()
+	var/mob/living/silicon/ai/M = pick_n_take(candidates)
```
Comment (Jared-Fogle) [inline code/game/gamemodes/dynamic/dynamic_rulesets_roundstart.dm:545]: Better variable name.

**Review by Cyberboss — APPROVED** [review]

**Comment by Jared-Fogle** [comment]
IT DOESNT WORK AND IT DOESNT EVEN HAVE AN ANTAG_CAP FIKOU

**Pull request merged**